app/users/admin/edit-user/edit-user.component.ts

```diff
@@ -4,6 +4,9 @@ import { ActivatedRoute, Router } from '@angular/router';
 import { Roles } from '../roles.model';
 import { User } from '../../user.model';
 import { UserService } from '../../user.service';
+import { error } from 'selenium-webdriver';
```

Comment:
What is this for? It's not even used here, as far as I can tell.

Reply:
I dont know where did this come from...removed too..